api/envoy/extensions/wasm/v3/wasm.proto

```diff
@@ -59,6 +59,7 @@ message VmConfig {
   // (proxy_on_start). `google.protobuf.Struct` is serialized as JSON before
   // passing it to the plugin. `google.protobuf.BytesValue` and
   // `google.protobuf.StringValue` are passed directly without the wrapper.
+  // [#protodoc-extension: envoy.wasm.runtime]
```

Comment:
the wasm plugins dont seem to have their own protos - i guess this is because the wasm runtime conf is passed as a  string through.

im wondering if there is any point to adding this here, the links it creates link to the runtime section immediately above and dont have additional config info

in terms of vscode/jsonschema, i guess it just needs to respect `Any`

Reply:
@lizan how come these differ from the usual story that each extension point needs to be typed. The choice of runtime should have a typed config, right?